app/routes/live.js

```diff
@@ -0,0 +1,12 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class LiveRoute extends Route {
+  @service router;
+
+  beforeModel(transition) {
+    if (transition?.to?.queryParams?.dev !== 'true') {
```

Comment:
I think the navigation should also be behind feature-flag
![image](https://github.com/Real-Dev-Squad/website-www/assets/78433013/ebae4cdd-8dd8-49b9-ac10-6fc3e5174829)